db/migrate/20160119092239_add_show_filter_to_custom_fields.rb

```diff
@@ -0,0 +1,6 @@
+class AddShowFilterToCustomFields < ActiveRecord::Migration
+  def change
+    add_column :custom_fields, :show_filter, :boolean, after: :sort_priority, default: false, null: false
```

Comment:
Good point. I'll go with the `search_filter` option. In Ruby context `acts_as_x` might be confusing because there are a lot of popular gems with that name.